packages/core/src/models/uischema.ts

```diff
@@ -245,3 +245,6 @@ export interface Categorization extends UISchemaElement {
 
 export const isGroup = (layout: Layout): layout is GroupLayout =>
   layout.type === 'Group';
+
+export const isLayout = (element: UISchemaElement): element is Layout =>
```

Comment:
We already have an `isLayout` function in `generators/`uischema`. Maybe you could unify both, such that the generator is then using the one exposed here.